python/lsst/ctrl/oods/msgIngester.py

```diff
@@ -115,12 +118,22 @@ async def ingest(self, butler_file_list):
         # for each butler, attempt to ingest the requested file,
         # Success or failure is noted in a message description which
         # will send out via a CSC logevent.
+        LOGGER.info("ingest called")
         try:
             for butler in self.butlers:
                 await butler.ingest(butler_file_list)
         except Exception as e:
             LOGGER.warning("Exception: %s", e)
 
+    def helper_done_callback(self, task):
```

Comment:
If you consider this method part of the class' public interface, please add a docstring. If it is supposed to be a private/internal thing, consider prefixing it with an underscore.

Reply:
Fixed.